#ifndef SEARCHNOTE_H
#define SEARCHNOTE_H

#include <QWidget>

namespace Ui {
class searchnote;
}

class searchnote : public QWidget
{
    Q_OBJECT

public:
    explicit searchnote(QWidget *parent = nullptr); // Konstruktor
    ~searchnote(); // Destruktor

private slots:
    void on_searchButton_searchnote_clicked(); // Slot f체r den Such-Button
    void on_backButton_searchnote_clicked(); // Slot f체r den Zur체ck-Button

private:
    Ui::searchnote *ui; // Benutzeroberfl채che
    void searchNotes(const QString &query); // Funktion zum Suchen von Notizen
};

#endif // SEARCHNOTE_H
